### DISCUSSION TITLE
Runtime Error

### DISCUSSION THREAD

**Discussion by GorkemP** [FETS-AI/Challenge#116] [Q&A] [answered]
Dear Organizers, 

We have just fetch latest commit however we got runtime error while we were running an experiment that works on the previous version.

The error is about reproducibility as shown below.

![image](https://user-images.githubusercontent.com/7123842/126807514-5a3e1cf7-47d0-4ec6-b1a3-070ccf18850d.png)

Thank you in advance.

**Comment by sarthakpati** [comment, marked as answer]
Hi,

Can you try putting this line 224 in `Task_1/openfl-workspace/fets_challenge_workspace/impl/fets_challenge_model.py`:

```python
torch.use_deterministic_algorithms(False)
```

**Reply by GorkemP** [reply]
After this change and installing packages again (`pip install .`), problem solved. Thank you.

**Reply by sarthakpati** [reply]
Brilliant. Can you now revert this change, reinstall and try again? If it doesn't work, please include this message in your submission Readme.
